packages/avatar/src/Avatar.stylesheet.js

```diff
@@ -8,14 +8,18 @@ const SizeMapping = {
   [sizes.XLARGE_64]: "extraLarge"
 };
 
+export function getBGColor(backgroundId, themeData) {
+  return themeData[`avatar.backgroundColor${backgroundId || 1}`];
+}
+
```

Comment:
Breaking the background color out as its own function as part of work to make the avatar background color accessible outside of the component

Reply:
Couldn't you change the background style color via the stylesheet function?

Reply:
I agree this function is not needed, I've removed it in a follow-up commit